tests/unit/resource/scheduler_binding_check.cpp

```diff
@@ -0,0 +1,86 @@
+#include <hpx/hpx_init.hpp>
+#include <hpx/async.hpp>
+#include <hpx/runtime/threads/executors/default_executor.hpp>
+#include <hpx/testing.hpp>
+#include <hpx/parallel/executors/thread_pool_executors.hpp>
```

Comment:
@biddisco I think you only need to rebase this on latest master to have the build errors go away (`hpx/parallel/executors/thread_pool_executors.hpp` not found). That said, you should also change the include to `hpx/execution/executors/thread_pool_executors.hpp`.